data/core/closetheloop.md

```diff
@@ -788,52 +708,7 @@
 * affirm
     - utter_explain_nlu
     - utter_explain_core
-    - utter_tryout
```

Comment:
shouldn't this be `utter_direct_step2`?

Reply:
it is, bit weirdly shown by the diff tool